src/partners/moonpay.ts

```diff
@@ -199,7 +199,13 @@ export const moonpay: PartnerPlugin = {
 }
 
 export function processMoonpayTx(rawTx: unknown): StandardTx {
-  const tx: MoonpayTx = asMoonpayTx(rawTx)
+  let tx: MoonpayTx
+  try {
+    tx = asMoonpayTx(rawTx)
+  } catch (e) {
+    console.log(e)
+    throw e
+  }
```

Comment:
`console.log` should be happening at the top of the call stack which means that this try..catch does nothing. If the console.log isn't happening at the top of the call-stack, then that's the bug.

Reply:
Yeah. this should be removed.

Reply:
it was added for debugging purposes.